update API and CLI docs to KES 2023-10-27T22-05-35Z (#38)
minio/kes-docs: content/cli/_index.md

| OS       | ARCH    | Binary                                                                                       |
|:--------:|:-------:|:--------------------------------------------------------------------------------------------:|
| linux    | amd64   | [linux-amd64](https://github.com/minio/kes/releases/latest/download/kes-linux-amd64)         |
| linux    | arm64   | [linux-arm64](https://github.com/minio/kes/releases/latest/download/kes-linux-arm64)         |
| darwin   | arm64   | [darwin-arm64](https://github.com/minio/kes/releases/latest/download/kes-darwin-arm64)       |
| windows  | amd64   | [windows-amd64](https://github.com/minio/kes/releases/latest/download/kes-windows-amd64.exe) |

> Download the binary via `curl` but replace `<OS>` and `<ARCH>` with your operating system and CPU architecture.

Reviewer: ```suggestion
Download the binary using `curl`.
Replace `<OS>` and `<ARCH>` with your operating system and CPU architecture.
```

## Available commands

|Command                                           |Description                            |

> |:-------------------------------------------------|:--------------------------------------|

Reviewer 2: @aead is it worth maintaining something like a deprecated API list for users on older KES versions?

We do something similar for the Server/MC docs where we keep deprecated commands + the version we dropped them from. It slightly simplifies support for users who are pinned to the older version.

cc @djwfyi

Reviewer 2: At a glance, we are dropping:

`enclave`
`manage`
`init`
